### PR TITLE
Explorer adding stake pool info

### PR DESCRIPTION
**Pull request by afposchenrieder**
This PR contains:

- Adding AntD table (more responsive, usefull for future offset-pagination in all tables. )
- Stake Pool search.
- Stake Pool component with produced blocks.

### CODE REVIEW

File: examples/explorer/README.md [hunk 1]
```diff
@@ -56,3 +57,24 @@ A simple Explorer using Reactjs, React-bootstrap and Relay to use with Jorgunman
...
+
+# Main directories
+
+- `data`: Here you can find the GraphQl schema of the server we are using, it should stay update with the server for Relay to work properly. Anyway, you can extend server's schema.
+- `src/graphql`: This directory contains objects related with Graphql Relay. This includes `environment`, `fetchData` and all queries that the application need to use when wrapping components.
+- `components/general`: This directory contains the three more general components in the app page.
+- `components/mainSection`: MainSection directory contains all the components that will be shown in the MainSection part of the page. This includes all search results and RecentBlocks components
+- `Address`, `Block`, `Certificate`, `Epoch`, `StakePool`, `Status` and `Transaction` directories contains components that shows information of each entity. For example, each directory contains an Info component that contains the basic entity information, and a FullEntityInfo components that contains more complex info.
+- `commons`: This directory holds multiple components usefull for all sections of the application.
```
Comment (enzoc4) [inline examples/explorer/README.md]: typo in 

> useful**l**

File: examples/explorer/src/components/Block/BlockTable/BlockTable.jsx [hunk 2]
```diff
@@ -1,63 +1,57 @@
...
+// Necesary because there is a problem with sort and paging with Antd table
+const sorter = (b1, b2) => Number(b2.chainLength) - Number(b1.chainLength);
+
+/**
+ * This function receives a boolean indicating if the table should be a
+ * CursorBasedTable. If not, it will return an OffsetBasedTable
+ * @param cursorType A boolean defining if the table should be a Cursor-based
+ * or an Offset-based pagination table.
+ */
+const BlockTable = ({ cursorType }) => {
```
Comment (enzoc4) [inline examples/explorer/src/components/Block/BlockTable/BlockTable.jsx]: we should definitely improve the backend to make this not needed

File: examples/explorer/src/components/Commons/CursorPagination/CursorPagination.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Pagination } from 'antd';
+import { TABLE_PAGE_SIZE } from '../../../helpers/constants';
+import './cursorPagination.scss';
+
+// Workaround to make Cursor based pagination work with Ant Table
```
Comment (enzoc4) [inline examples/explorer/src/components/Commons/CursorPagination/CursorPagination.jsx]: This refers to that comparison below?
` if (page < current) { `

Comment (afposchenrieder) [reply examples/explorer/src/components/Commons/CursorPagination/CursorPagination.jsx]: Yep, but also, to the whole component. i'll change the comment to make it more clear

**Pull request merged**